src/currencyconverterapi.js

```diff
@@ -0,0 +1,55 @@
+const checkAndPush = (isoCc, ccArray) => {
+  if (isoCc !== 'iso:USD' && isoCc.slice(0, 4) === 'iso:') {
+    const cc = isoCc.slice(4).toUpperCase()
+    if (!ccArray.includes(cc)) {
+      ccArray.push(cc)
+    }
+  }
+}
+
+export const currencyconverterapiPlugin = {
+  pluginType: 'exchange',
+
+  makePlugin ({ io }) {
+    return Promise.resolve({
+      exchangeInfo: {
+        exchangeName: 'CurrencyConverterAPI'
+      },
+
+      async fetchExchangeRates (pairsHint) {
+        // TODO: pairsHint is broken in Core and doesn't actually look at wallet or account
+        // fiat settings. Hard code known rare currencies for now
+        pairsHint = [
+          { fromCurrency: 'iso:USD', toCurrency: 'iso:IMP' },
+          { fromCurrency: 'iso:USD', toCurrency: 'iso:IRR' }
+        ]
+        const isoCodesWanted = []
+        for (const pair of pairsHint) {
+          checkAndPush(pair.fromCurrency, isoCodesWanted)
+          checkAndPush(pair.toCurrency, isoCodesWanted)
+        }
+
+        const pairs = []
+        for (const isoCode of isoCodesWanted) {
```

Comment:
We could probably refactor this into `Promise.all(isoCodesWanted.map(...))` for parallelism, but this seems fine for now, especially since we have a short list.